jetpack-beta.php

```diff
@@ -180,6 +180,8 @@ public function remove_activate_stable( $actions ) {
 	public function update_all_plugins( $plugins ) {
 		// WP.com requests away show regular plugin
 		if ( defined( 'REST_API_REQUEST' ) && REST_API_REQUEST ) {
+			//Ensure that Jetpack reports the version it's using on account of the Jetpack Beta plugin to Calypso
+			$plugins[ JETPACK_PLUGIN_FILE ]['Version'] = $plugins[ JETPACK_DEV_PLUGIN_FILE ]['Version'];
```

Comment:
I think we need to do a bit more checking what version is currently active. And then use that to display the right version info and not always dev versions info.

so something like 

```
if ( is_plugin_active( JETPACK_DEV_PLUGIN_FILE ) ) {
 $plugins[ JETPACK_PLUGIN_FILE ]['Version'] = $plugins[ JETPACK_DEV_PLUGIN_FILE ]['Version'];
}

Reply:
Wouldn't we only be in that file if the plugin is active? And if the plugin is active, wouldn't that dictate that we'd use the dev version?

Reply:
nope that is not always the case. The Jetpack beta could be active and the user could be running the regular version of jetpack